app/src/main/java/org/simple/clinic/home/report/ReportsScreen.kt

```diff
@@ -5,40 +5,24 @@ import android.content.Context
 import android.os.Parcelable
 import android.util.AttributeSet
 import android.widget.FrameLayout
-import com.jakewharton.rxbinding3.view.detaches
 import io.reactivex.Observable
-import io.reactivex.rxkotlin.ofType
 import kotlinx.android.synthetic.main.screen_report.view.*
-import org.simple.clinic.ReportAnalyticsEvents
-import org.simple.clinic.bindUiToController
 import org.simple.clinic.di.injector
 import org.simple.clinic.mobius.MobiusDelegate
 import org.simple.clinic.util.unsafeLazy
-import org.simple.clinic.widgets.ScreenCreated
-import org.simple.clinic.widgets.ScreenDestroyed
-import org.simple.clinic.widgets.UiEvent
 import org.simple.clinic.widgets.visibleOrGone
 import javax.inject.Inject
 
 class ReportsScreen(context: Context, attrs: AttributeSet) : FrameLayout(context, attrs), ReportsUi {
 
-  @Inject
-  lateinit var controller: ReportsScreenController
-
   @Inject
   lateinit var effectHandler: ReportsEffectHandler
 
-  private val events by unsafeLazy {
-    screenCreates()
-        .compose(ReportAnalyticsEvents())
-        .share()
-  }
-
   private val delegate by unsafeLazy {
     val uiRenderer = ReportsUiRenderer(this)
 
     MobiusDelegate.forView(
-        events = events.ofType(),
+        events = Observable.empty(),
```

Comment:
Can we use `Observable.never` instead? `Observable.empty()` immediately signals the completion of the stream, which is not what an actual event stream behaves like (never completes).

Reply:
Okay, I wasn't sure what was the difference between these 2 in this context. So went with `empty`